httphandlers/buildinfo_handler.go

```diff
@@ -3,12 +3,21 @@ package httphandlers
 import (
 	"encoding/json"
 	"github.com/Financial-Times/service-status-go/buildinfo"
+	"github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
 	"net/http"
 )
 
-// BuildInfoHandler uses the buildinfo.GetBuildInfo to respond to requests for build info !
-func BuildInfoHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Any reason for the renaming the function? Only asking as this is now a breaking change to anyone using this project.